src/components/Chat.js

```diff
@@ -69,7 +73,8 @@ const Chat = (props) => {
     setMessage(e.target.value);
   }
   function handleSubmit() {
-    if (yourName != "" && selectedIcon != null) {
+    //changed yourName to check for type
```

Comment:
This comment better serves as a message for the commit/PR, rather than being in the source code. If this change were to be implemented as is, this comment would not make sense.